test.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+echo "*** Running active_model_validators specs"
+
+bundle install         || exit 1
+bundle exec rspec spec || exit 1
```

Comment:
No need to use `|| exit 1` here, if tests will fail it will automatically exit with code 1